test/test-server.coffee

```diff
@@ -11,9 +11,18 @@ class Server
   options: {}
 
   constructor: (options = {}) ->
+    app.set('view engine', 'ejs')
+    app.set('views', __dirname)
+    app.get '/', (req, res) ->
+      res.render('index', {env: process.env.NODE_ENV})
+
     @options.port = options.port || @defaults.port
     @options.host = options.host || @defaults.host
     app.listen(@options.port)
-    console.log("[TEST_SERVER] Server running at: #{@options.host}/#{@options.port}")
 
-module.exports = new Server()
+    if (!isTest)
```

Comment:
Maybe we just don't auto-instantiate a server here? Have that happen in the caller, instead?